javascript/javascript1/week1/housy-pricey.js

```diff
@@ -0,0 +1,23 @@
+let volumeInMeters=8*10*10;
+let gardenSizeInM2=100;
+let peterHouseCost=2500000;
+
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+console.log(housePrice);
+if (housePrice < peterHouseCost) {
+    console.log("peter is paying too much"); 
+} else {
+    console.log("peter is paying too little");
+}
+
+
+volumeInMeters=5*11*8;
+gardenSizeInM2=70;
+juliaHouseCost= 1000000;
```

Comment:
there is no variable declared with the name of `juliaHouseCost`
I think you forget to declare it before assigning the value like:
```js 
var juliaHouseCost = 1000000;
```
Note that the code works and JavaScript ignores this error. 
But it is better to declare variable names always